packages/core/src/components/button/_button.scss

```diff
@@ -68,6 +68,21 @@ Styleguide components.button.css
     }
   }
 
+  &.pt-loading {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
flex seems a little over-the-top here when we're going to hide all the real content and absolutely position the spinner. might `display: flex` also end up changing the size of certain complex buttons?